src/components/EntityPage/File/index.js

```diff
@@ -331,6 +333,24 @@ const FileEntity = ({ api, fileId, isPageLoading, hasFilePermission }) => (
                   }}
                   loading={file.isLoading}
                   data={toParticpantBiospecimenData(data)}
+                  transforms={{
+                    study_name: studyShortName => (
+                      <ExternalLink
+                        href={`${kfWebRoot}/support/studies-and-access`}
+                        onClick={e => {
+                          trackUserInteraction({
+                            category: TRACKING_EVENTS.categories.entityPage.file,
+                            action:
+                              TRACKING_EVENTS.actions.click +
+                              `: Associated Participants/Biospecimens: Study Name`,
+                            label: studyShortName,
+                          });
+                        }}
+                      >
+                        {studyShortName}
+                      </ExternalLink>
+                    ),
+                  }}
```

Comment:
@ciaranschutte this actually looks good to me, I like this so we can keep data separate from presentation. The bug was caused by React components being mixed in with the data that gets pulled into the export file :)